v3/1upkeyboards/1upslider8/1upslider8.json

```diff
@@ -0,0 +1,132 @@
+{
+    "name": "1upslider8",
```

Comment:
Perhaps rename this to 1UP Slider8?
(this shows up on the VIA website)

Reply:
We were trying to stay consistent with naming conventions, and also avoid using spaces since you may not use spaces in qmk names

Reply:
This name isn't reflected in "builds". it shows up on the website and whats in the top right corner of VIA